Docker/ods-api-bulk-load-console/alpine/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN dotnet tool install -g \
     rm -f /root/.dotnet/tools/.store/edfi.suite3.bulkloadclient.console/$BULKLOAD_VERSION/edfi.suite3.bulkloadclient.console/$BULKLOAD_VERSION/tools/net8.0/any/log4net.config
 
 # Smaller image
-FROM mcr.microsoft.com/dotnet/runtime:8.0.2-alpine3.19@sha256:17c0535218406fa12788a26925c6d31e27cb52edf94c1aa7666f33ab804b1663
+FROM mcr.microsoft.com/dotnet/runtime:8.0.3-alpine3.19@sha256:1d1ddab7d571bb995f8a6e9e7f7b441d0c1a2b3a12a42ad0fda085bb4e8e9e94
 RUN apk --upgrade --no-cache add gettext=~0 bash=~5 && \
```

Comment:
Vulnerabilities in this image can be fixed by adding `busybox=~1.36 openssl=~3.1`
For the Npgsql issue, update EdFi.Suite3.Common version to 7.2.337 in the Loadtools projects